pkg/maps/authmap/auth_map.go

```diff
@@ -61,8 +62,8 @@ func newMap(maxEntries int) *authMap {
 // LoadAuthMap loads the pre-initialized auth map for access.
 // This should only be used from components which aren't capable of using hive - mainly the Cilium CLI.
 // It needs to initialized beforehand via the Cilium Agent.
-func LoadAuthMap() (Map, error) {
-	bpfMap, err := ebpf.LoadRegisterMap(MapName)
+func LoadAuthMap(logger *slog.Logger) (Map, error) {
```

Comment:
Rather than add parameters to these initializers to reference the logger, I think we would prefer to have such dependencies injected as part of a Hive cell.

Reply:
I agree but AFAIK, the `LoadAuthMap` is called from cilium-dbg which is not converted to use cells.